packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html

```diff
@@ -231,3 +231,53 @@ <h4 class="pt-3">
     </form>
     <p data-translate="telephony_line_softphone_save_theme"></p>
 </oui-collapsible>
+<oui-collapsible
+    class="custom-collapsible-no-click mt-3"
+    data-expanded="true"
+    data-heading="{{ 'telephony_line_softphone_logo_title' | translate }}"
+>
+    <form name="form" data-ng-submit=":: $ctrl.applyTheme()">
+        <div class="row">
+            <oui-field
+                data-ng-class="{
+                                    'col-md-10': $ctrl.logoFilename!=='',
+                                    'col-md-12': $ctrl.logoFilename===''
```

Comment:
What about `null` or `undefined` values ?

Reply:
because api is like this :)

For instance to delete you have to /put with empty strings ;)